scripts/pi-hole/php/header.php

```diff
@@ -416,80 +416,80 @@ function pidofFTL()
                 <!-- Home Page -->
                 <li<?php if($scriptname === "index.php"){ ?> class="active"<?php } ?>>
                     <a href="index.php">
-                        <i class="fa fa-fw menu-icon fa-home"></i> Dashboard
+                        <i class="fa fa-fw menu-icon fa-home"></i> <span>Dashboard</span>
                     </a>
                 </li>
                 <?php if($auth){ ?>
                 <!-- Query Log -->
                 <li<?php if($scriptname === "queries.php"){ ?> class="active"<?php } ?>>
                     <a href="queries.php">
-                        <i class="fa fa-fw menu-icon fa-file-alt"></i> Query Log
+                        <i class="fa fa-fw menu-icon fa-file-alt"></i> <span>Query Log</span>
                     </a>
                 </li>
                 <li class="treeview<?php if($scriptname === "db_queries.php" || $scriptname === "db_lists.php" || $scriptname === "db_graph.php"){ ?> active<?php } ?>">
                   <a href="#">
-                    <i class="fa fa-fw menu-icon fa-clock"></i> Long-term data
+                    <i class="fa fa-fw menu-icon fa-clock"></i> <span>Long-term data</span>
                     <span class="pull-right-container">
                       <i class="fa fa-angle-left pull-right"></i>
                     </span>
                   </a>
                   <ul class="treeview-menu">
                     <li<?php if($scriptname === "db_graph.php"){ ?> class="active"<?php } ?>>
                         <a href="db_graph.php">
-                            <i class="fa fa-fw menu-icon fa-file-alt"></i> Graphics
+                            <i class="fa fa-fw menu-icon fa-file-alt"></i> <span>Graphics</span>
```

Comment:
**Note 1:**
The `<span>` tags are necessary only for main items (ex: line 431).
They are not necessary for sub-items, inside `<ul class="treeview-menu">` (but it's not a problem).

**Note 2:** 
accidentally, I added this tags to #2040.

Reply:
I added it for all of them intentionally to hopefully avoid forgetting about them again in the future. How does one *accidentally* do this?

Reply:
> How does one _accidentally_ do this?

I was testing the other PR and saw the strange menu. I tried to fix it and forgot to revert the changes before send the PR.